app/models/saved_claim.rb

```diff
@@ -176,6 +176,17 @@ def regional_office
     []
   end
 
+  # Required for Lighthouse Benefits Intake API submission
+  # Subclasses can override to provide alternate metadata if needed
+  def metadata_for_benefits_intake
+    address = parsed_form['claimantAddress'] || parsed_form['veteranAddress'] || {}
+    { veteranFirstName: parsed_form.dig('veteranFullName', 'first'),
+      veteranLastName: parsed_form.dig('veteranFullName', 'last'),
+      fileNumber: parsed_form['vaFileNumber'] || parsed_form['veteranSocialSecurityNumber'],
+      zipCode: address['postalCode'],
```

Comment:
`zipCode` is a required metadata field - if this isn't found then `BenefitsIntake::Metadata#validate_zip_code(metadata)` will fail and the submission will ultimately fail. 

If address['postalCode'] returns `nil` (which is possible here) then the submission will bomb out. This will need to be fixed.

Reply:
I don't think it should be up to our team to alter behavior for saved claims beyond our team's. This should have the same success rate as the previous code.

The same could've been said for app/sidekiq/lighthouse/submit_benefits_intake_claim.rb:79

Reply:
Take a look [here as well](https://github.com/department-of-veterans-affairs/vets-api/blob/1124e3929cf6d8551e34a83e41243c8c61d798a2/lib/lighthouse/benefits_intake/metadata.rb#L96-L106) as well - this is what ultimately handles the zip code validation. If you read the comment, it says that if a zip code doesn't conform to regex, then we need to set to 5 zeroes. So I would argue that we aren't introducing a new behavior here, but rather reinforcing the one that already exists and ensuring that any non-zipcode whether it be foreign, malformed, or nonexistent, will be set to a default string of `"00000"`